src/components/ProfileMatch/index.tsx

```diff
@@ -124,24 +125,33 @@ export default function ProfileMatch({
         renderIconGroup(timeCommitmentMatch as MatchField<Listing>)}
       {listingData.listing_type !== 'INT' &&
         renderIconGroup(startDateMatch as MatchField<Listing>)}
-      <Flex $align="center" $gap="16px">
-        <Box $h="16px" $w="16px" $textAlign="center">
-          {((listingData.listing_type === 'CASE' && !interpretation) ||
-            listingData.listing_type === 'LCA') &&
-          matchedLanguages.length === 0 ? (
-            <Icon type="yellowExclamation" />
-          ) : (
-            matchIcon(matchedLanguages.length > 0)
-          )}
-        </Box>
-        <P>
-          You {matchedLanguages.length > 0 ? '' : "don't "}list{' '}
-          {matchedLanguages.length > 0
-            ? formatEnumeration(matchedLanguages, 'and')
-            : formatEnumeration(listingData.languages, 'or')}{' '}
-          in your languages.
-        </P>
-      </Flex>
+      {listingData.languages.length > 0 ? (
```

Comment:
again, i understand why we're handling the case when the listings do not have any languages. but are we requiring languages for each listing?